src/Yoti.Auth/CryptoEngine.cs

```diff
@@ -58,7 +58,7 @@ public static byte[] DecryptRsa(byte[] cipherBytes, AsymmetricCipherKeyPair keyp
         public static byte[] SignDigest(byte[] digestBytes, AsymmetricCipherKeyPair keypair)
         {
             // create a signature from the digest using SHA256 hashing with RSA
-            var signer = SignerUtilities.GetSigner(DigestAlgorithm);
+            ISigner signer = SignerUtilities.GetSigner(DigestAlgorithm);
```

Comment:
But here we've changed it the other way

Reply:
So it's only if the explicit type `ISigner` is used on the right hand side, that we use var. Although you could guess that it would be of type `ISigner`, it isn't explicit